src/pages/Login.js

```diff
@@ -4,6 +4,8 @@ import { useNavigate } from "react-router-dom";
 export default function Login() {
   const navigate = useNavigate();
 
+  localStorage.removeItem(`user`);
```

Comment:
This is another side effect that is being triggered every time the component renders